README.md

```diff
@@ -40,9 +40,15 @@ Before running the demo app, you need to download and install the project depend
 yarn install
 ```
 
-## Run
+## Pre-Setup iOS
+
+Carthage is also needed for fetching the Aztec dependency and you can get it running following command:
```

Comment:
I don't think `preios` will *install* Carthage. I think we just need to add Carthage to the "Prerequisites" section

Reply:
Thanks @koke , you are right. It confused me for some reason.
I have added the "Prerequisites" section. Please, take a look.